tests/unit/Makefile.am

```diff
@@ -13,11 +13,13 @@ noinst_PROGRAMS = \
 	deque \
 	freelist \
 	msgbuff \
+	show_tuner_decisions \
 	scheduler \
 	idpool
 
 TESTS = $(noinst_PROGRAMS)
 
+show_tuner_decisions_SOURCES = show_tuner_decisions.c $(top_builddir)/src/tuner/nccl_ofi_tuner.c $(top_builddir)/src/tuner/nccl_ofi_model.c
```

Comment:
This is an anti-pattern in Automake, and that's probably why Autoconf is getting angry about `AM_PROG_CC_C_O` (that and AL2 is ancient).  What you should do is add a helper library, similar to what we're doing on the LDADD on line 8.